samtranslator/translator/translator.py

```diff
@@ -198,6 +199,7 @@ def prepare_plugins(plugins):
     """
 
     required_plugins = [
+        DefaultDefinitionBodyPlugin(),
```

Comment:
Do we have a default naming convention? Half of these are camel case, the other half use underscores. Do we need to update our documentation for naming conventions?

Reply:
This is Python convention. `DefaultDefinitionBodyPlugin` is a class; others are functions/methods.

Reply:
Ok, thank you for the clarification.